client-frameworks-support/client-support-angular/projects/client-support-angular/src/lib/luigi-mock/luigi-mock.module.ts

```diff
@@ -1,172 +1,14 @@
 import { APP_INITIALIZER, NgModule } from '@angular/core';
+import { LuigiMockEngine } from '../../../../../../testing-utilities/dist/';
```

Comment:
`import { LuigiMockEngine } from '@luigi-project/testing-utilities';`

add this later. keep duplicate temporarily below